setup.py

```diff
@@ -238,7 +255,7 @@ def getcompilecommand(self):
         def chop(self, pathname):
             """returns the path relative to self.dist_dir"""
             assert pathname.startswith(self.dist_dir)
-            return pathname[len(self.dist_dir):]
+            return pathname[len(self.dist_dir) :]
```

Comment:
Why it would not be legal? For example, PEP-8 says "in a slice the colon acts like a binary operator", so if there is space used around binary operators in other cases, it should be here as well.

Reply:
I never saw whitespace around colons in slices, but you are right that PEP8 allows it. Nevertheless I dislike it.